ui/perfherder/compare/CompareSubtestDistributionView.jsx

```diff
@@ -157,6 +167,11 @@ export default class CompareSubtestDistributionView extends React.Component {
             <LoadingSpinner />
           ) : (
             <Row className="justify-content-center mt-4">
+              <Col className="text-center">
+                {this.state.failureMessages.length !== 0 && (
+                  <ErrorMessages errorMessages={this.state.failureMessages} />
```

Comment:
Whenever you are using the same state variable twice, use it like this
const {failureMessages} = this.state;
And then write just failureMessage instead of this.state.failureMessage